opsdroid/connector/github/tests/test_connector_github.py

```diff
@@ -411,19 +444,23 @@ async def test_skill(opsdroid, config, event):
             data=get_webhook_payload("github_issue_close_payload.json"),
         )
         assert resp.status == 201
+        assert "Test skill complete" in caplog.text
+        assert "Exception when running skill" not in caplog.text
 
 
 @pytest.mark.add_response(
     "/user", "GET", get_response_path("github_user.json"), status=200
 )
 @pytest.mark.asyncio
-async def test_receive_label(opsdroid, connector, mock_api):
+async def test_receive_label(opsdroid, connector, mock_api, caplog):
     """Test a receive label event creates a message and parses it."""
+    caplog.set_level(logging.INFO)
 
-    @match_event(github_event.Unlabeled)
+    @match_event(github_event.Labeled)
     async def test_skill(opsdroid, config, event):
-        assert event.labels == "bug"
+        assert len(event.labels) == 1 and event.labels[0]["name"] == "bug"
```

Comment:
Had to change the way this assertion worked for now because the labels are handled inconsistently for labeling and unlabeling. I have a suggestion for this, but will do the work in a separate small PR.